modules/media/php/hidefilefilter.class.inc

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * This file provides an implementation of the hide files filter.
+ *
+ * PHP Version 7
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\media;
+define('SHOW', 0);
```

Comment:
This shouldn't be here. We're not using PHP4. (Also defining "SHOW" as 0 is a really weird and confusion convention.

Reply:
As in it shouldn't be declared using `define`? or that we should have class constants at all?

Reply:
it should
1. use a class `const`, not `define()` (or maybe just a namespaced const if it doesn't make sense on theclass)
2. use a value other than "0" as the definition (since 0 is traditionally more falsey than truey and it's confusing to have Show be 0.

Reply:
I could set the `HIDE` to `0` and have the function `return $resource->getHideFile() !== HIDE;` if that makes more sense.

Reply:
I could set the value of `HIDE` to `1` and have the function `return $resource->getHideFile() !== HIDE;` if that makes more sense.

Reply:
@driusan This was my change, so my bad. The idea of using `0` was so that it would match the back-end value for the `hile_file` column which, I agree, should probably be `show_file` which would make truthy/falsey-ness less confusing.